example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Transport library for sending HTTP requests and opening WebSockets.
 homepage: https://github.com/Workiva/w_transport
 
 environment:
-  sdk: ">=2.11.0 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
This subpackage isn't actually ready to be migrated because it depends on over_react. Will have to revert this change.